src/k-provider/ovp/response-types/kaltura-user-entry.ts

```diff
@@ -45,7 +44,7 @@ export class KalturaUserEntry {
    * @constructor
    * @param {Object} entry The json response
    */
-  constructor(entry: Object) {
+  constructor(entry: any) {
```

Comment:
align Object and any.
also, lint is yelling- **every member in this file is missing** `public`